### PR TITLE
SPLICE-1987: Add fully qualified table name into HBASE "TableDisplayName" Attribute

### PR DESCRIPTION
**Pull request by stpabhi**

### CODE REVIEW

**Comment by stpabhi** [comment]
@jleach4 @wtang16 @dgomezferro @jyuanca please review

File: hbase_storage/hbase1.2.0/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java [hunk 1]
```diff
@@ -55,8 +55,9 @@ public PartitionCreator withName(String name){
 
     @Override
     public PartitionCreator withDisplayNames(String[] displayNames){
```
Comment (dgomezferro) [inline hbase_storage/hbase1.2.0/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java]: Please update the corresponding HPartitionCreator for the other platforms (hbase1.0.0, hbase1.1.0...)

Comment (stpabhi) [reply hbase_storage/hbase1.2.0/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java]: Ok

Comment (stpabhi) [reply hbase_storage/hbase1.2.0/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java]: @dgomezferro Done, added to the other platforms.

File: hbase_storage/hbase1.0.0.x/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java [hunk 2]
```diff
@@ -55,6 +55,7 @@ public PartitionCreator withName(String name){
 
     @Override
     public PartitionCreator withDisplayNames(String[] displayNames){
+        descriptor.setValue(SIConstants.SCHEMA_DISPLAY_NAME_ATTR, displayNames[0]);
         descriptor.setValue(SIConstants.TABLE_DISPLAY_NAME_ATTR, displayNames[0] != null ? displayNames[0] : descriptor.getNameAsString());
```
Comment (dgomezferro) [inline hbase_storage/hbase1.0.0.x/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java]: In the other platforms you forgot to update (increment by 1) the displayNames indexes for the TABLE/INDEX attributes.

Comment (stpabhi) [reply hbase_storage/hbase1.0.0.x/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java]: Oops, will make those changes

Comment (stpabhi) [reply hbase_storage/hbase1.0.0.x/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java]: @dgomezferro Done, Thanks for reviewing.

**Pull request merged**